route/route.go

```diff
@@ -337,6 +382,108 @@ func (r *Router) batch(w http.ResponseWriter, req *http.Request) {
 	w.Write(response)
 }
 
+func (r *Router) Export(ctx context.Context, req *collectortrace.ExportTraceServiceRequest) (*collectortrace.ExportTraceServiceResponse, error) {
+
+	md, ok := metadata.FromIncomingContext(ctx)
+	if !ok {
+		r.Logger.Error().Logf("Unable to retreive metadata from OTLP request.")
+		return &collectortrace.ExportTraceServiceResponse{}, nil
+	}
+
+	var requestID types.RequestIDContextKey
+	debugLog := r.iopLogger.Debug().WithField("request_id", requestID)
```

Comment:
It might be helpful to have a note here about why requestID is blank.

Reply:
Added comment in [eaaaa52](https://github.com/honeycombio/refinery/pull/202/commits/eaaaa5227dbc67f04cd07fe283bba3d1ffabe4a0).